shell/download/README.md

```diff
@@ -0,0 +1,61 @@
+# Dataverse Dataset Downloader
+
+## Purpose
+
+The Dataverse Dataset Downloader is a shell script that automates the process of downloading files from a Dataverse dataset and organizing them into their respective folders.
+It's particularly useful when the dataset has many files and/or a directory structure than needs to be preserved.
+
+## Prerequisites
+
+- Bash shell (Linux or macOS)
+- `wget`
+- `grep`
+- `sed`

```

Comment:
Does this work well on Mac? (I think @landreev may have tested it there.) I ask because it uses BSD sed which can be a bit different than GNU sed.